### PR TITLE
Rc vs 63 vat sop documentation

### PR DESCRIPTION
**Pull request by RoriCremer**

### CODE REVIEW

File: scripts/variantstore/variant_annotations_table/README.md [hunk 1]
```diff
@@ -54,12 +52,7 @@ To grab any files not in the bucket, but in the file of file names:
 
 _(dont forget to do this for the indices as well)_
 
-
-
-
-There is a line of code in ExtractAnAcAfFromVCF (the most expensive in $ and time task in the workflow) that can be removed because it is used to track the variants that are dropped _TODO: Rori to make using it a parameter_
-
-
+All dropped variants are tracked
```
Comment (rsasch) [inline scripts/variantstore/variant_annotations_table/README.md:55]: Can you add a bit more detail as to how they are tracked?

File: scripts/variantstore/wdl/GvsCreateVATAnnotations.wdl [hunk 2]
```diff
@@ -136,6 +136,7 @@ task ExtractAnAcAfFromVCF {
         echo_date "VAT: normalize, left align and split multi allelic sites to new lines, remove duplicate lines"
         bcftools norm --threads 4 -m- --check-ref w -f Homo_sapiens_assembly38.fasta filtered.bcf -O b -o normalized.bcf
         rm filtered.bcf
+        ## note that normalization may create sites with more than 50 alt alleles
```
Comment (rsasch) [inline scripts/variantstore/wdl/GvsCreateVATAnnotations.wdl:139]: Seems like this comment should be up a couple of lines?

**Review by rsasch — CHANGES_REQUESTED** [review]
Unfortunately have to leave these notes here, since the text in question isn't part of the PR diff; apologies if my meaning isn't clear.

> They are cleaned up after 24 hours, but this code needs to be tweaked so that you can’t get into a state where duplicates are created here. The real question here is going to be, is there a use case that we might want to run where adding to a VAT that was created say weeks ago is beneficial, but given that calculations occur on a sample summing level, this seems unlikely.

Do we still need this?  Seems like it belongs in a ticket (or two) and not documentation.

> To check that all of the shards have successfully made it past the first of the sub workflow steps (the most complicated and likely to fail) they will need to be annotated / transformed into json files and put here: gsutil ls [output_path]/annotations/ | wc -l

This (mostly "they will need to be annotated / transformed into json files and put here") makes it sound like the person running the WDL needs to do something else, instead of this being a way to check on the success of the WDL.

Lastly, is the TSV file the only deliverable?  Is there anything else that needs to be communicated...etc.?  Either way, can this be explicitly added to this doc?

**Review by gbggrant — APPROVED** [review]

**Review by rsasch — CHANGES_REQUESTED** [review]
Sorry to harp on this.  I think you should change
> The pipeline takes in a jointVCF and outputs a table in BigQuery.

to something like

> The pipeline takes in jointVCFs (and index files), creates a table in BigQuery, and outputs a bgzipped TSV file containing the contents of that table, which is the only deliverable.

to make it clear what the final "thing" is that is created and delivered to the client/user.

**Review by rsasch — APPROVED** [review]
looks good 👍🏻

**Pull request merged**